src/melee.cpp

```diff
@@ -1369,14 +1393,6 @@ bool player::block_hit(Creature *source, body_part &bp_hit, damage_instance &dam
     // So that we don't suddenly switch that for any reason.
     const bool weapon_blocking = can_weapon_block();
     if( weapon_blocking ) {
-        int block_bonus = 2;
-        if (shield.has_technique( WBLOCK_3 )) {
-            block_bonus = 10;
-        } else if (shield.has_technique( WBLOCK_2 )) {
-            block_bonus = 6;
-        } else if (shield.has_technique( WBLOCK_1 )) {
-            block_bonus = 4;
-        }
```

Comment:
Right here where you cut that large section that is now the body of `blocking_ability`.
We need to have `block_bonus` set, so that `block_score` below can use it. Otherwise it will use the default `block_score` which is just 1.